pkg/branding/brandedtext_test.go

```diff
@@ -45,10 +45,9 @@ func (s *BrandedTextTestSuite) TestGetBrandedProductName() {
 			productBrandingEnv: "STACKROX_BRANDING",
 			brandedProductName: brandedProductNameStackrox,
 		},
-		// TODO(ROX-10208): Change this to StackRox after changing the default value of ProductBrandingEnvName.
 		"Unset env": {
 			productBrandingEnv: "",
-			brandedProductName: brandedProductNameRHACS,
```

Comment:
You need to rebase from recent `master` to see `TestGetBrandedProductNameShort` in this file and make the same change there.